src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -31,6 +31,8 @@ namespace System.Collections
 #if FEATURE_NETCORE
     [FriendAccessAllowed]
 #endif
+    /// <summary>Implements the <see cref="T:System.Collections.IList" /> interface using an array whose size is dynamically increased as required.To browse the .NET Framework source code for this type, see the Reference Source.</summary>
```

Comment:
Also, why do we need that last sentence about browsing the Reference Source?  Is that actually in the MSDN docs, or was that added automatically by the tool?  Seems quite odd, given that a) you're in the source, and b) a tool that uses this to produce docs can add such a message automatically when producing the docs... it shouldn't be in the source.